src/components/editMeasure/details/MeasureDefinitions/MeasureDefinitions.tsx

```diff
@@ -72,7 +85,105 @@ const MeasureDefinitions = (props: MeasureDefinitionsProps) => {
     }
   }, [setMeasureDefinitions, measure]);
 
-  const handleSubmit = (values: MeasureDefinition) => {};
+  // Toast utilities
+  const [toastOpen, setToastOpen] = useState<boolean>(false);
+  const [toastMessage, setToastMessage] = useState<string>("");
+  const [toastType, setToastType] = useState<string>("danger");
+  const onToastClose = () => {
+    setToastType("danger");
+    setToastMessage("");
+    setToastOpen(false);
+  };
+  const handleToast = (type, message, open) => {
+    setToastType(type);
+    setToastMessage(message);
+    setToastOpen(open);
+  };
+  //first sort by term then by definition
+  const sortDefinitions = (
+    definitions: MeasureDefinition[]
+  ): MeasureDefinition[] => {
+    const sortFunction = (a: MeasureDefinition, b: MeasureDefinition) => {
+      const term1 = a.term ? a.term.toLowerCase() : "";
+      const term2 = b.term ? b.term.toLowerCase() : "";
+      const definition1 = a.definition.toLowerCase();
+      const definition2 = b.definition.toLowerCase();
+      if (term1 < term2) {
+        return -1;
+      }
+      if (term2 > term1) {
+        return 1;
+      }
```

Comment:
are these if conditions identical? I don't think second if condition ever be hit

Reply:
@adongare Yeah sorry typo. Just corrected the mistake, thanks Ashok!